src/timesync/timesyncd-manager.c

```diff
@@ -825,21 +826,23 @@ int manager_connect(Manager *m) {
                         bool restart = true;
 
                         /* Our current server name list is exhausted,
-                         * let's find the next one to iterate. First
-                         * we try the system list, then the link list.
-                         * After having processed the link list we
-                         * jump back to the system list. However, if
-                         * both lists are empty, we change to the
-                         * fallback list. */
+                         * let's find the next one to iterate. First we try the runtime list, then the system list,
+                         * then the link list. After having processed the link list we jump back to the
+                         * system list, then to the runtime list. However, if all lists are empty, we change to
```

Comment:
"jump back to the runtime list, then to the system list if no runtime server exists."